hydra-cluster/exe/hydra-cluster/Main.hs

```diff
@@ -25,7 +24,6 @@ run options =
       case knownNetwork of
         Just network ->
           withCardanoNodeOnKnownNetwork fromCardanoNode workDir network $ \node -> do
-            waitForFullySynchronized fromCardanoNode node
```

Comment:
why do we need to remove this?

Reply:
Now this is part of running the cardano-node and we wait for the sync before continuing further